Realm/RLMSyncUser.h

```diff
@@ -245,6 +267,41 @@ NS_SWIFT_UNAVAILABLE("Use the full version of this API.");
 /// :nodoc:
 + (instancetype)new __attribute__((unavailable("RLMSyncUser cannot be created directly")));
 
-NS_ASSUME_NONNULL_END
+@end
+
+/**
+ A data object representing information about a user that was retrieved from a user lookup call.
+ */
+@interface RLMSyncUserInfo : NSObject
+
+/**
+ The authentication provider which manages the user represented by this user info instance.
+ */
+@property (nonatomic, readonly) RLMIdentityProvider provider;
+
+/**
+ The provider-based username of the user.
+ */
+@property (nonatomic, readonly) NSString *providerID;
```

Comment:
Is this a username or identity? The doc comment suggests a username, but the property name indicates otherwise. It'd also be good to be consistent about whether we use `identity` or `ID` in these property names. We're using both within this same class.